### PR TITLE
Generate multiple experiment  in batch for load testing

### PR DESCRIPTION
**Pull request by dhavalocked**

### CODE REVIEW

File: packages/Upgrade/src/api/controllers/ExperimentController.ts [hunk 1]
```diff
@@ -370,6 +370,37 @@ export class ExperimentController {
...
+   *    post:
+   *       description: Generate New Experiments
+   *       consumes:
+   *         - application/json
+   *       parameters:
+   *         - in: body
+   *           name: experiments
+   *           required: true
+   *           schema:
+   *             type: Array of Experiment object
```
Comment (VivekFitkariwala) [inline packages/Upgrade/src/api/controllers/ExperimentController.ts]: Is this working for Swagger?

File: packages/Upgrade/src/api/controllers/ExperimentController.ts [hunk 2]
```diff
@@ -370,6 +370,37 @@ export class ExperimentController {
...
+   *         - Experiments
+   *       produces:
+   *         - application/json
+   *       responses:
+   *          '200':
+   *            description: New Experiment is created
+   */
+
+  @Post('/batch')
+  public createTestExperiments(
```
Comment (VivekFitkariwala) [inline packages/Upgrade/src/api/controllers/ExperimentController.ts]: createTestExperiments -> createMultipleExperiments || createExperiments

File: packages/Upgrade/src/api/controllers/ExperimentController.ts [hunk 3]
```diff
@@ -370,6 +370,37 @@ export class ExperimentController {
...
+    @Body({ validate: { validationError: { target: false, value: false } } }) experiment: Experiment[]
+  ): Promise<Experiment[]> {
+    return this.experimentService.createTestExperiments(experiment);
```
Comment (VivekFitkariwala) [inline packages/Upgrade/src/api/controllers/ExperimentController.ts]: Change this name also

File: packages/Upgrade/src/api/services/ExperimentService.ts [hunk 4]
```diff
@@ -109,6 +109,11 @@ export class ExperimentService {
     return this.addExperimentInDB(experiment, currentUser);
   }
 
+  public createTestExperiments(experiment: Experiment[]): Promise<Experiment[]> {
```
Comment (VivekFitkariwala) [inline packages/Upgrade/src/api/services/ExperimentService.ts]: Change this name to more generic

File: packages/Upgrade/src/api/controllers/ExperimentController.ts (hunk 3 above)
Comment (dhavalocked) [reply packages/Upgrade/src/api/controllers/ExperimentController.ts]: renaming createTestExperiments to  createMultipleExperiments

**Pull request merged**